examples/plot-graph-using-bokeh.ipynb

```diff
@@ -0,0 +1,200 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "65682475",
+   "metadata": {},
+   "source": [
+    "# Plot graph using `bokeh`\n",
+    "\n",
+    "#### Topics covered in this example\n",
+    "* Installing python libraries on the EMR cluster.\n",
+    "* Use Bokeh plotting library to plot trignometric functions: sin, cos and tan functions. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "92b7c5ca",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "This is an example from [bokeh tutorial](https://github.com/bokeh/jupyter_bokeh/blob/main/examples/jupyter_interactors.ipynb). In this example, we are going to use [Bokeh](https://docs.bokeh.org/en/latest/docs/user_guide/jupyter.html) to plot an interactive graph. We will plot different trignometric functions, namely sin, cos and tan. To interactive with the graph, we will use a dropdown that offers a choice to select a function and the sliders to control the frequency, amplitude, and phase."
```

Comment:
This doesn't work when the file is converted to html for the links. You will have to use href, example can be found [here](https://raw.githubusercontent.com/aws-samples/emr-studio-notebook-examples/main/examples/install-notebook-scoped-libraries-at-runtime.ipynb).